101-streamanalytics-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "streamAnalyticsJobName": {
+      "value": "StreamAnalyticsJobName"
```

Comment:
The name has to be unique only within the subscription, not world wide globally, do I still need to change this?